examples/growthcurve.py

```diff
@@ -100,65 +106,57 @@ def __call__ (self, r):
 
 def main():
 
-    date = datetime.datetime.now().strftime("%a %Y-%m-%d %H:%M:%S")
-
     ########################################################################
     # command line arguments and options
     ########################################################################
 
-    parser = optparse.OptionParser(usage = __doc__)
-    #parser.add_option("-a", "--aa", dest="aa", type=float,
+    parser = optparse.OptionParser(usage=__doc__)
+    # parser.add_option("-a", "--aa", dest="aa", type=float,
     #                  default=1.0, help="default=%default")
 
     opts, args = parser.parse_args()
 
     if len(args) == 0:
         r1, r2, dr = 3.0, 3.0, 0.5
     elif len(args) == 3:
-        r1, r2, dr = map(float, args)
+        r1, r2, dr = list(map(float, args))
     else:
         parser.print_help()
         sys.exit(1)
 
-
     # make a list of radii to compute the growthcurve points
     radius = []
-    nR = int( (r2 - r1)/dr + 1 )
+    nR = int((r2 - r1)/dr + 1)
     for iR in range(nR):
         radius.append(r1 + iR*dr)
 
-
     # make an image big enough to hold the largest requested aperture
     xwidth = 2*(0 + 128)
     ywidth = xwidth
 
     # initializations
     sigmas = [1.5, 2.5]  # the Gaussian widths of the psfs we'll use
     nS = len(sigmas)
-    a = 100.0
-    aptaper = 2.0
-    xcen = xwidth/2
-    ycen = ywidth/2
     alg = measBase.PsfFluxAlgorithm
     schema = afwTable.SourceTable.makeMinimalSchema()
     schema.addField("centroid_x", type=float)
     schema.addField("centroid_y", type=float)
     plugin = alg(measBase.PsfFluxControl(), "test", schema)
     cat = afwTable.SourceCatalog(schema)
     cat.defineCentroid("centroid")
-    print "# sig rad  Naive Sinc Psf"
+    print("# sig rad  Naive Sinc Psf")
     for iS in range(nS):
-        sigma = sigmas[iS];
+        sigma = sigmas[iS]
 
         # make a Gaussian star to measure
-        gauss  = afwMath.GaussianFunction2D(sigma, sigma)
+        gauss = afwMath.GaussianFunction2D(sigma, sigma)
         kernel = afwMath.AnalyticKernel(xwidth, ywidth, gauss)
-        kimg   = afwImage.ImageD(kernel.getDimensions())
+        kimg = afwImage.ImageD(kernel.getDimensions())
         kernel.computeImage(kimg, False)
-        kimg  *= 100.0
-        mimg   = afwImage.MaskedImageF(kimg.convertFloat(),
-                                       afwImage.MaskU(kimg.getDimensions(), 0x0),
-                                       afwImage.ImageF(kimg.getDimensions(), 0.0))
+        kimg *= 100.0
+        mimg = afwImage.MaskedImageF(kimg.convertFloat(),
+                                     afwImage.MaskU(kimg.getDimensions(), 0x0),
```

Comment:
Nicely said. I think it was intended to emphasize that this is a mask, whose values are usually set as hex values.